packages/vue-components/src/Dropdown.vue

```diff
@@ -120,58 +132,58 @@ export default {
     showDropdownMenu() {
       this.show = true;
       $(this.$refs.dropdown).findChildren('ul').each(ul => ul.classList.toggle('show', true));
-    }
+    },
   },
-  mounted () {
-    const $el = $(this.$refs.dropdown)
+  mounted() {
+    const $el = $(this.$refs.dropdown);
     if (this.show) {
       this.showDropdownMenu();
     }
-    $el.onBlur((e) => { this.hideDropdownMenu() }, false)
-    $el.findChildren('a,button.dropdown-toggle').on('click', e => {
-      e.preventDefault()
+    $el.onBlur(() => { this.hideDropdownMenu(); }, false);
+    $el.findChildren('a,button.dropdown-toggle').on('click', (e) => {
+      e.preventDefault();
       if (this.disabledBool) { return false; }
-      if (this.showBool) {
+      if (this.show) {
         this.hideDropdownMenu();
       } else {
         this.showDropdownMenu();
       }
       return false;
-    })
-    $el.findChildren('ul').on('click', 'li>a', e => { 
-      if (e.target.classList.contains('submenu-toggle')) { return }
+    });
+    $el.findChildren('ul').on('click', 'li>a', (e) => {
+      if (e.target.classList.contains('submenu-toggle')) { return; }
       this.hideDropdownMenu();
-    })
+    });
+  },
+  beforeDestroy() {
+    const $el = $(this.$refs.dropdown);
+    $el.offBlur();
+    $el.findChildren('a,button').off();
+    $el.findChildren('ul').off();
   },
-  beforeDestroy () {
-    const $el = $(this.$refs.dropdown)
-    $el.offBlur()
-    $el.findChildren('a,button').off()
-    $el.findChildren('ul').off()
-  }
-}
+};
 </script>
 
 <style scoped>
-.secret {
-  position: absolute;
-  clip: rect(0 0 0 0);
-  overflow: hidden;
-  margin: -1px;
-  height: 1px;
-  width: 1px;
-  padding: 0;
-  border: 0;
-}
+    .secret {
```

Comment:
looks good!

just one nit: let's enable and fix stylelint separately (for all `.vue` files that are es-linted)

Reply:
Sure 👍 
Should I do it in this PR or open a separate one?

Reply:
let's do it separately, iirc none of the .vue files are validated in our ci tests - might be quite a few changes

Reply:
Alright. I've enabled stylelint for all `.vue` files but added all of them to `.stylelintignore` except for `dropdown.vue` for now. Will do it separately for all the other files in the next PR :)